environment.yml

```diff
@@ -1,6 +1,7 @@
 name: mne
 channels:
-- defaults
+- conda-forge/label/vtk_dev
```

Comment:
Why this channel? `vtk` is in `conda-forge` with its latest release version 9.0.1.

Reply:
Yes, this shouldn't be needed anymore.

Reply:
I agree it seems like it shouldn't be necessary but in practice it was necessary to get 3.8 to install

Reply:
Maybe we can try to figure out why - the fewer channels we use the less problems we will run into. Ideally, we should only use the `conda-forge` channel.

Reply:
Feel free to dig into it with the `conda` folks, in the meantime we have been using this for the `server_environment.yml` for a while so I'm happy living with it until `conda` fixes it